### PR TITLE
REFS maintenance - consumer role fix.

### PR DESCRIPTION
**Pull request by ZASMan**
### Master Pivotal Ticket
* https://www.pivotaltracker.com/n/projects/2481179/stories/179070524

### CODE REVIEW

File: app/controllers/insured/consumer_roles_controller.rb [hunk 1]
```diff
@@ -249,7 +249,7 @@ def update
   end
 
   def ridp_agreement
-    set_current_person
+    set_current_person unless current_user&.has_hbx_staff_role?
     consumer = @person.consumer_role
```
Comment (bill-transue) [inline app/controllers/insured/consumer_roles_controller.rb:253]: What will `@person` be if we're not running `set_current_person`?

**Review by bill-transue — CHANGES_REQUESTED** [review]
We can not just make things run when we encounter a nil. We need to back and figure out what either what user with an hbx_staff_role doesn't have a person. Or why `set_current_person` isn't working with admin impersonation.

File: app/controllers/insured/consumer_roles_controller.rb (hunk 1 above)
Comment (ZASMan) [reply app/controllers/insured/consumer_roles_controller.rb:253]: https://github.com/ideacrew/enroll/blob/trunk/app/views/insured/consumer_roles/ridp_agreement.html.erb Don't think its a big deal cuz RIDP agreement is just some text without variables but we could revert it if necessary

**Review by bill-transue — APPROVED** [review]
Zane explained it to me and this makes more sense now. Let's QA it. Thanks @ZASMan!

**Pull request merged**